app/views/my_facilities/missed_visits.html.erb

```diff
@@ -16,11 +16,7 @@
         <p class="c-grey-dark c-print-black">
           <strong>Numerator:</strong> <%= t("missed_visits_copy.numerator") %>
           <strong>Denominator:</strong>
-          <% if current_admin.feature_enabled?(:report_with_exclusions) %>
-            <%= t("denominator_excluding_ltfu_copy", region_name: "each facility") %>
-          <% else %>
-            <%= t("denominator_without_exclusions_copy", region_name: "each facility") %>
```

Comment:
Can this YML key be removed from the locale now?

Reply:
Addressed in 4c9b2f3.